Kudu.Services/Diagnostics/ProfileManager.cs

```diff
@@ -18,16 +18,24 @@ internal static class ProfileManager
     {
         private const int ProcessExitTimeoutInSeconds = 180;
 
+        private const string SERVICE_PROFILER_AGENT_GUID = "F5091AA9-80DC-49FF-A7CF-BD1103FE149D";
```

Comment:
Please rename all const names to Pascal casing like the rest of Kudu files. like ServiceProfilerAgentGuid